docs/ru/mkdocs.yml

```
@@ -37,7 +37,7 @@ plugins:
     data: data
 nav:
 - FastAPI: index.md
-- Languages:
+- Языки:
```

```suggestion
- Languages:
```

Yep, agreed with @mbroton ...also, the scripts that generate the docs depend on this key being `Languages`, so I need to keep it there before merging. :sweat_smile: